src/overlays/actors/ovl_En_Ma1/z_en_ma1.c

```diff
@@ -71,7 +74,7 @@ static AnimationFrameCountInfo sAnimationInfo[] = {
     { &gMalonChildSingAnim, 1.0f, ANIMMODE_LOOP, -10.0f },
 };
 
-static Vec3f D_80AA16B8 = { 800.0f, 0.0f, 0.0f };
+static Vec3f sMultVec = { 800.0f, 0.0f, 0.0f };
```

Comment:
you can leave this unnamed, it's not super helpful

it's the position in limb space (more specifically, I assume the head limb) of the actor's focus position (i.e. the position link targets)
imo these should be named like `focus[Limb]LocalPos`, like `focusHeadLocalPos`, but that requires knowing what limb they use for the transform, so a lot of assets docs

anyway this data can be moved into the function if the mouth and eye textures arrays are also moved down. It'll probably be done eventually, or you can

Reply:
I see, I just noticed something similar was done in another actor so I just followed suit. I'll revert it 👍

Reply:
the stuff he mentions about moving the variables would look like this.

Basically, data that gets referenced in only 1 function can be moved into that function. And we usually prefer this so you dont have to go looking around the file for the data. You just need to make sure to preserve the datas order in the file.
The order is still vec -> mouth textures -> eye textures.

If you dont want to mess with doing it this way thats fine, but I think itd be good to have if youre up for it
```c
void EnMa1_PostLimbDraw(PlayState* play, s32 limbIndex, Gfx** dList, Vec3s* rot, void* thisx) {
    EnMa1* this = (EnMa1*)thisx;
    Vec3f vec = { 800.0f, 0.0f, 0.0f };

    if (limbIndex == 15) {
        Matrix_MultVec3f(&vec, &this->actor.focus.pos);
    }
}

void EnMa1_Draw(Actor* thisx, PlayState* play) {
    static void* sMouthTextures[] = {
        gMalonChildNeutralMouthTex,
        gMalonChildSmilingMouthTex,
        gMalonChildTalkingMouthTex,
    };
    static void* sEyeTextures[] = {
        gMalonChildEyeOpenTex,
        gMalonChildEyeHalfTex,
        gMalonChildEyeClosedTex,
    };
    EnMa1* this = (EnMa1*)thisx;
    Camera* activeCam;
    f32 distFromCamEye;
    s32 pad;

    OPEN_DISPS(play->state.gfxCtx, "../z_en_ma1.c", 1226);
}
```